scripts/kibana-proxy.py

```diff
@@ -48,46 +39,60 @@ class KibanaProxy:
+                                          detach=True,
+                                          environment={
+                                              'AWS_ACCESS_KEY_ID': creds.access_key,
+                                              'AWS_SECRET_ACCESS_KEY': creds.secret_key,
+                                              'AWS_SESSION_TOKEN': creds.token,
+                                              'AWS_REGION': os.environ['AWS_DEFAULT_REGION']
+                                          },
+                                          network='host',
+                                          ports={port: port for port in (kibana_port, cerebro_port, proxy_port)},
+                                          tty=True)
+            self.containers.append(proxy)
             time.sleep(3)
-            self.spawn('docker', 'run', '--rm', '-t',
-                       '--network', f'container:{container_name}',
-                       '-e', f'ELASTICSEARCH_HOSTS=http://localhost:{proxy_port}',
-                       '-e', f'SERVER_PORT={kibana_port}',
-                       'docker.elastic.co/kibana/kibana-oss:6.8.0')
-            self.spawn('docker', 'run', '--rm', '-t',
-                       '--network', f'container:{container_name}',
-                       'lmenezes/cerebro:0.8.5',  # 0.9.1 does not work against ES 6.8.0
-                       f'-Dhttp.port={cerebro_port}')
+            kibana = client.containers.run('docker.elastic.co/kibana/kibana-oss:6.8.0',
+                                           auto_remove=True,
+                                           detach=True,
+                                           environment={
+                                               'ELASTICSEARCH_HOSTS': f'http://localhost:{proxy_port}',
+                                               'SERVER_PORT': kibana_port
+                                           },
+                                           network_mode=f'container:{proxy.name}',
+                                           tty=True)
+            self.containers.append(kibana)
+            # 0.9.1 does not work against ES 6.8.0
+            cerebro = client.containers.run('lmenezes/cerebro:0.8.5',
+                                            auto_remove=True,
+                                            command=[f'-Dhttp.port={cerebro_port}'],
+                                            detach=True,
+                                            network_mode=f'container:{proxy.name}',
+                                            tty=True)
+            self.containers.append(cerebro)
             time.sleep(10)
             print(f'Now open Kibana at http://127.0.0.1:{kibana_port}/')
-            print(f'Now open Cerebro at http://127.0.0.1:{cerebro_port}/ and paste http://localhost:{proxy_port}')
-            self.wait()
+            print(f'Now open Cerebro at http://127.0.0.1:{cerebro_port}/ '
+                  f'and paste http://localhost:{proxy_port}')
```

Comment:
Be aware that you can avoid the manual/copy paste using
```suggestion
            print(f'Now open Cerebro at http://127.0.0.1:{cerebro_port}/#/overview?host=http://localhost:{proxy_port}')
```
But there might be some other use-case I'm not aware of where the user would want to specify a different host.

Reply:
This is great, thanks! I was looking for something like this but wasn't able to find it.